install.sh

```diff
@@ -238,6 +239,13 @@ __install_zeek() {
                 "security:zeek" \
                 "https://download.opensuse.org/repositories/security:/zeek/xUbuntu_$(lsb_release -rs)/Release.key"
                 ;;
+            Debian)
+                __install_packages cmake make gcc g++ flex bison libpcap-dev libssl-dev python3 python3-dev python3-git python3-semantic-version swig zlib1g-dev gpg
+                __add_deb_repo  "deb http://download.opensuse.org/repositories/security:/zeek/$_Debian_Release/ /" \
```

Comment:
Since we probably won't need to access the variable elsewhere in the code, I would prefer if we made the Debian_Release variable a local variable in this section of the switch statement. Most repositories will use the OS codename rather than the tags the opensuse build service is using.

Reply:
Understood. I moved this to a local scope and retested it.